docs/src/administration_backup_and_restore.md

```diff
@@ -37,6 +37,41 @@ cat db_dump.sql  |  docker exec -i FOLDERNAME_postgres_1 psql -U lemmy # restore
 docker exec -i FOLDERNAME_postgres_1 psql -U lemmy -c "alter user lemmy with password 'bleh'"
 ```
 
+### Changing your domain name
+
+If you haven't federated yet, you can change your domain name in the DB. **Warning: do not do this after you've federated, or it will break federation.**
+
+```
+-- Post
```

Comment:
I would add that you need to run those in `docker-compose exec -u lemmy postgres psql` (or something like that). I would also change the lines above to use docker-compose command instead of docker, its just simpler.